### DISCUSSION TITLE
I have been approved for the student benefits But still not getting any confirmation mail

### DISCUSSION THREAD

**Discussion by Avinash-Codes** [community/community#109893] [GitHub Education] [answered]
### Select Topic Area

Question

### Body

i haven't receive any mail regarding the github student pack nor got any benefit please help

![Screenshot 2024-02-24 182318](https://github.com/community/community/assets/133222313/cacc386e-25fc-4945-a3a8-22f634ccebe3)

its been more than three days

**Comment by eyadamr905** [comment]
how can i order it i don't know?
I'm student too

**Reply by Avinash-Codes** [reply]
go to this website and apply with your college or institution email id
https://education.github.com/pack

**Comment by queenofcorgis** [comment, marked as answer]
Thanks for posting in the GitHub Community, @AVINASH-CODER-PRO ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
